manifests/kubernetes-prometheusRule.yaml

```diff
@@ -719,31 +699,6 @@ spec:
       for: 15m
       labels:
         severity: critical
-  - name: kubernetes-system-scheduler
-    rules:
-    - alert: KubeSchedulerDown
-      annotations:
-        description: KubeScheduler has disappeared from Prometheus target discovery.
-        runbook_url: https://github.com/prometheus-operator/kube-prometheus/wiki/kubeschedulerdown
-        summary: Target disappeared from Prometheus target discovery.
-      expr: |
-        absent(up{job="kube-scheduler"} == 1)
-      for: 15m
-      labels:
-        severity: critical
-  - name: kubernetes-system-controller-manager
-    rules:
-    - alert: KubeControllerManagerDown
-      annotations:
-        description: KubeControllerManager has disappeared from Prometheus target
-          discovery.
-        runbook_url: https://github.com/prometheus-operator/kube-prometheus/wiki/kubecontrollermanagerdown
-        summary: Target disappeared from Prometheus target discovery.
-      expr: |
-        absent(up{job="kube-controller-manager"} == 1)
-      for: 15m
-      labels:
-        severity: critical
```

Comment:
This **shouldn't** be removed from kube-prometheus at all. We already include [an addon](https://github.com/prometheus-operator/kube-prometheus/blob/main/jsonnet/kube-prometheus/platforms/kops.libsonnet#L17-L31) to enable gathering data from kcm and scheduler, the one lacking thing is documenting how to configure kops to allow kcm and scheduler to bind to `0.0.0.0` and not `127.0.0.1` so metrics can be gathered. That said documentation should be done in separate PR.